spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/internal/spring/ArmeriaConfigurationUtil.java

```diff
@@ -365,18 +372,21 @@ public static void configureAnnotatedHttpServices(
                                  .addAll(bean.getRequestConverters())
                                  .addAll(bean.getResponseConverters())
                                  .build();
+            final String serviceName = bean.getService().getClass().getName();
             docServiceRequests.addAll(bean.getExampleRequests());
+            docServiceHeaders.put(serviceName, bean.getExampleHeaders());
             server.annotatedService(bean.getPathPrefix(), bean.getService(), decorator,
                                     exceptionHandlersAndConverters);
 
             if (!Strings.isNullOrEmpty(docsPath)) {
                 docServiceRequests.forEach(
-                        exampleReq -> {
-                            final String serviceName = bean.getService().getClass().getName();
-                            docServiceBuilder.exampleRequestForMethod(serviceName,
-                                                                      exampleReq.getMethodName(),
-                                                                      exampleReq.getExampleRequest());
-                        });
+                        exampleReq ->
+                                docServiceBuilder.exampleRequestForMethod(serviceName,
+                                                                          exampleReq.getMethodName(),
+                                                                          exampleReq.getExampleRequest()));
+                for (Entry<String, Collection<HttpHeaders>> entry : docServiceHeaders.entrySet()) {
```

Comment:
Could use forEach here as well? e.g.

```java
docServiceHeaders.forEach(docServiceBuilder::exampleHttpHeaders);
```

Reply:
I fixed this and `configureThriftServices (...)` as well.